SpywareFilter/sections/allowlist.txt

```diff
@@ -36,6 +36,8 @@
 !@@||mc.yandex.ru/metrika/watch.js$domain=
 !#endif
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/197918#issuecomment-2649307173
+@@||js.braintreegateway.com/web/$script,domain=fiverr.com
```

Comment:
It seems it is paypal's antifraud system and all site affected
![image](https://github.com/user-attachments/assets/4b967359-ce28-4f49-81f1-380eb02d8b2d)

Removed `||js.braintreegateway.com/web/*/js/data-collector.min.js`

@piquark6046 Please be careful when blocking analytics on complicated sites.